about.html

```diff
@@ -130,6 +142,7 @@ <h2>Information Technology</h2>
             </div>
         </footer>
     </div>
+    <script src="Js/modal_header.js"></script>
```

Comment:
Acho que vc pode adicionar o atributo `defer` e mover esse script pra a linha 12 depois do `import` dos estilos.
```suggestion
    <script src="Js/modal_header.js" defer></script>
```

Reply:
Feito, isso é pra o js carregar primeiro, né isso?

Reply:
Isso é pra o javascript esperar a pagina carregar e os elements html estarem prontos pra depois carregar o javascript.